app/controllers/customer_resources_controller.rb

```diff
@@ -13,8 +13,7 @@ def show
 
   def update
     @customer_resource.update customer_resource_params
```

Comment:
In AR, a collection can update a single record via the id. This seems like less magic, and more in keeping with convention. Is there anything preventing this here?

```ruby
@resource = customer_resources.update customer_resource_id, customer_resource_params
render jsonapi: @resource
```

Reply:
It's not how flexirest works out of the box: https://github.com/flexirest/flexirest/blob/a4b89653b672cdffb3f084f37bbd81ee77c09afa/Ruby-on-Rails-Integration.md#controller

Not sure it's worth the extra steps to get that syntax.